docker-wrappers/DOMINO/Dockerfile

```
@@ -0,0 +1,10 @@
+# DOMINO wrapper
+# https://github.com/Shamir-Lab/DOMINO
+FROM python:3.7
+
+RUN pip install domino-python
```

```suggestion
RUN pip install domino-python==0.1.1
```
We'll pin a specific version of the package in case they release updates